src/SearchResults.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import moment from "moment";
+
+const SearchResults = props => {
+  return (
+    <div>
+      <table className="table">
+        <thead>
+          <tr>
+            <th scope="">Id</th>
+            <th scope="col">Title</th>
+            <th scope="col">First Name</th>
+            <th scope="col">Surname</th>
+            <th scope="">Email</th>
+            <th scope="col">Room Id</th>
+            <th scope="col">Check-In-Date</th>
+            <th scope="col">Check-Out-Date</th>
+            <th scope="col">Number of Night/s</th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.results.map((field, index) => {
+            const nightsSpent = moment(field.checkOutDate).diff(
+              field.checkInDate,
+              "days"
+            );
+            return (
+              <tr key={index}>
```

Comment:
You could extract this into a separate component, e.g. `BodyRow`.

As a general rule, if you are mapping through something and returning a piece of JSX each time, then you can extract that JSX into a separate component.